pages/index.tsx

```diff
@@ -25,9 +27,9 @@ export default function HomePage() {
       title="Home"
     >
       <div className="w-full max-w-4xl mx-auto mt-32 md:mt-56">
-        <h1 className="text-xl md:text-2xl lg:text-3xl">
-          Search all R packages on CRAN and Bioconductor
-        </h1>
+        <div className="text-xl md:text-2xl lg:text-3xl">
+          {`Search all ${packageCount>0?packageCount:''} R packages on CRAN and Bioconductor`}
```

Comment:
would be nice to add some spaces so that it's easier to read 👍 
```suggestion
          Search all {packageCount > 0 ? `${packageCount} ` : ''}R packages on CRAN and Bioconductor
```